### DISCUSSION TITLE
favicon missing on all pages

### DISCUSSION THREAD

**Discussion by nicolgit** [mmistakes/minimal-mistakes#3490] [Q&A] [answered]
### What happened?

- Minimal Mistakes version: jan 2022
- Ruby gem or remote theme version:
- Jekyll version: N/A
- Git repository URL: https://github.com/nicolgit/nicolgit.github.io
- Hosted on GitHub Pages (if yes provide URL to site): https://nicolgit.github.io
- Operating system: windows 11

I have added favicon.png in the main folder of the repo.

I have also updated _includes/head.html :   https://github.com/nicolgit/nicolgit.github.io/commit/1501dd8bc805d1e94a0798d60dce4022b33880ff

the result is that if i open https://nicolgit.github.io I see the favicon on the browser tab, I am not able to see the same icon if I open any blog post (i.e. https://nicolgit.github.io/azure-hub-and-spoke-playground/)

### Expected behavior

see the favicon icon ALSO on any blog post

### Steps to reproduce the behavior

N/A

**Comment by mmistakes** [comment, marked as answer]
Your `favicon` path isn't correct, that's why it doesn't show up any of your post/pages and only the home.
You need to make it relative, which can easily be done by adding a preceding `/` or using Jekyll's `relative_url` filter.

Change [your favicon line](<link rel="shortcut icon" type="image/png" href="favicon.png">) to this:

```html
<link rel="shortcut icon" type="image/png" href="{{ 'favicon.png' | relative_url }}">
```

**Reply by nicolgit** [reply]
problem resolved with your help, thank you!

**Reply by thinkhuman** [reply]
After a lot of experimenting and failing, I found that it should be (for me):
`<link rel="shortcut icon" type="image/png" href="{{ '/assets/images/favicon.png' | relative_url }}">`

**NOT:**
`<link rel="icon" type="image/png" href="{{ '/assets/images/favicon.png' | relative_url }}">`
